lms/djangoapps/learner_dashboard/utils.py

```diff
@@ -0,0 +1,19 @@
+""" The utility methods and functions to help the djangoapp logic
+"""
+from urlparse import urlparse
+
+
+FAKE_COURSE_ID_FOR_URL = 'course/replace+1'
+
+
+def url_remove_fake_course_id(url_string):
```

Comment:
Reversing with a fake ID, then stripping it out, doesn't seem like the cleanest way to do this, but I can't think of something better right now. That said, this helper is too complicated. More concisely:
```py
def strip_course_id(url):
    return url.split(FAKE_COURSE_ID)[0]
```